x-pack/platform/plugins/shared/onechat/server/routes/tools.ts

```diff
@@ -227,7 +229,62 @@ export function registerToolsRoutes({ router, getInternalServices, logger }: Rou
         });
       })
     );
+  // bulk delete tools
+  router.versioned
+    .post({
+      path: '/api/chat/tools/_bulk_delete',
```

Comment:
do we really want to expose this tools-specific _bulk_delete endpoint as public, it's not part of the onechat api spec

maybe keep this as internal endpoint, @pgayvallet @joemcelroy wdyt>